### ISSUE TITLE
calloc - reverse order of parameters

### ISSUE THREAD

**Issue by Geist-zz** [amakukha/tmg#3]
calloc (libs.h) has reverse order of parameters - size first, then number of elements.
Google search for calloc gives the reverse order (I guess it's not a big deal if it works despite that).

**Comment by amakukha** [comment]
True. Thanks. Though the order doesn't matter really, it might be neat to fix it in the future.
